src/index.ts

```diff
@@ -0,0 +1,607 @@
+import * as fs from "fs";
+import FormData from "form-data";
+import { spawn, execSync } from "child_process";
+import { ArgumentParser } from "argparse";
+import { ApiScanFormat, FormTypes, OnFailure, ScanMode, SubmitActions } from "./utils/enums";
+import { exit } from "process";
+import SOOSAnalysisApiClient from "@soos-io/api-client/dist/api/SOOSAnalysisApiClient";
+import {
+  getEnvVariable,
+  isUrlAvailable,
+  convertStringToB64,
+  sleep,
+} from "@soos-io/api-client/dist/utilities";
+import {
+  ScanStatus,
+  ScanType,
+  soosLogger,
+  LogLevel,
+  OutputFormat,
+  SOOS_CONSTANTS,
+} from "@soos-io/api-client";
+import { ZAPCommandGenerator, CONSTANTS } from "./utils";
+
+export interface SOOSDASTAnalysisArgs {
+  apiKey: string;
+  apiURL: string;
+  appVersion: string;
+  branchName: string;
+  branchUri: string;
+  buildUri: string;
```

Comment:
we should be consistent with the casing of existing parameters
```suggestion
  branchURI: string;
  buildURI: string;
```